newsroom/reports/reports.py

```diff
@@ -379,6 +371,18 @@ def get_product_company():
     return results
 
 
+def get_companies_id_by_product(product_id):
```

Comment:
This might be better placed in `newsroom.companies.companies`, in case it might be needed elsewhere later